DESCRIPTION

```diff
@@ -23,7 +23,7 @@ License: MIT + file LICENSE
 URL: https://github.com/r-lib/evaluate
 BugReports: https://github.com/r-lib/evaluate/issues
 Depends:
-    R (>= 3.0.2)
+    R (>= 4.0.0)
```

Comment:
The ~5 years on our policy seems to be sufficient for everyone (or at least we haven't had any complaints). And since we don't test versions of R older than that, we've discovered that it's very easy to accidentally depend on  features introduced in newer features, meaning that without tests, I don't trust the version declaration.

Reply:
Understood.  Following the CRAN release, I'll do the change in knitr and rmarkdown then. Thanks !